openstudiocore/src/model_editor/InspectorGadget.cpp

```diff
@@ -648,6 +648,9 @@ void InspectorGadget::layoutText( QVBoxLayout* layout,
       if(prop.minBoundType != IddFieldProperties::Unbounded)
       {
         double d = *(prop.minBoundValue);
+        if(prop.minBoundType == IddFieldProperties::ExclusiveBound) {

```

Comment:
@kbenne it is possible that unit conversion could round this epsilon out, it would be better to add minInclusive and minExclusive methods on the widget.  This is probably ok for most cases though